examples/notebook/pytorch/Quick_Started_Notebook_of_INC_for_Pytorch.ipynb

```diff
@@ -272,58 +332,134 @@
     "\n",
     "So far, we can finally start to quantize the model. \n",
     "\n",
-    "To start, we need to set the configuration for post-training quantization using `PostTrainingQuantConfig` class. Once the configuration is set, we can proceed to the next step by calling the `quantization.fit()` function. This function performs the quantization process on the model and will return the best quantized model."
+    "To start, we need to set the configuration for post-training quantization using `get_default_static_config()` to get static quant config. Once the configuration is set, we can proceed to the next step by calling the `prepare`, `convert` function. This function performs the quantization process on the model and will return the best quantized model."
```

Comment:
We got the best quantized model only when using `autotune`. Here we quantize the model based on the user-provided config.